README.md

```diff
@@ -1,3 +1,7 @@
 # parcel-examples
 
 🛠 Parcel examples using various frameworks
+
```

Comment:
Ideally, the readme links to the local examples in the repo for discoverability.

Reply:
- [x] Done in 60c675aae5a609881a99102e11015936880c79da